test/libsolidity/smtCheckerTests/try_catch/try_4.sol

```diff
@@ -0,0 +1,28 @@
+pragma experimental SMTChecker;
+
+abstract contract D {
+	function d() external virtual;
+}
+
+contract C {
+
+	int x;
+	D d;
+
+	function set(int n) public {
+		x = n;
+	}
+
+	function f() public {
+		x = 0;
+		try d.d() {
+			assert(x == 0); // should fail, x can be anything here
```

Comment:
Can you also add a test similar to this, where
- function `set` has a `require(n < 100);` before the assignment
- the assert here is `assert(x < 100);` which should hold

Reply:
Added.